docs/roles/ruby.md

```diff
@@ -7,3 +7,10 @@ command if a Gemfile is found in the project root directory.
 
 * **ruby_version**: this should be the exact version name (such as 2.3.3).
 Find a list of accepted version with `rbenv install -l`. Default is 2.4.0
+
+# Rails
+
+Will install `rails` with `rbenv` (see Ruby role) if listed in `Gemfile`.
+Fallback to normal installation with `gem` otherwise.
```

Comment:
What's the purpose of this? Why not always use the provided `Gemfile` and let the user add the rails dependency to this file? It seems overkill to me to create a role that just runs `gem install rails`.